model/llm/prompt/parse.go

```diff
@@ -13,16 +14,27 @@ var (
 )
 
 // ParseResponse parses code from a model's response.
-func ParseResponse(response string) (code string) {
+func ParseResponse(response string) (assessment metrics.Assessments, code string) {
+	assessment = metrics.Assessments{}
+
 	// Some models produce duplicated code tags, so unify them if needed.
 	response = codeTagDuplicatedRe.ReplaceAllString(response, "```")
```

Comment:
We slowly are rewording our "Re" to Match internally can you do -> `codeTagDuplicatedMatch`. At this point i am wondering if "Pattern" is better btw.